homeassistant/components/sensor/pan_tilt_phat.py

```diff
@@ -0,0 +1,82 @@
+"""
+Support for pan-tilt pHAT.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/sensor.pan_tilt_phat
+"""
+
+from datetime import timedelta
+import logging
+
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['pantilthat==0.0.4']
+_LOGGER = logging.getLogger(__name__)
+DOMAIN = 'pan_tilt_phat'
+SCAN_INTERVAL = timedelta(seconds=1)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the pan-tilt sensor."""
+    try:
+        import pantilthat
+    except OSError:
+        _LOGGER.error("No pan-tilt pHAT was found.")
+        return False
+
+    def home_service(call):
+        """Return the stage to home."""
+        pantilthat.pan(0)
+        pantilthat.tilt(0)
+        return True
+
+    def pan_service(call):
+        """Pan the stage."""
+        pantilthat.pan(call.data.get("Pan"))
+        return True
+
+    def tilt_service(call):
+        """Pan the stage."""
+        pantilthat.tilt(call.data.get("Tilt"))
+        return True
+
+    hass.services.register(DOMAIN, 'home', home_service)
```

Comment:
This is not allowed. You are not having a DOMAIN, you are under the sensor component. This should not be a platform for the sensor component but instead be it's own domain.